src/lib/constants.ts

```diff
@@ -4,20 +4,12 @@ export const keywords = [
   "Help",
   "List:",
   ",",
-  "Sessions",
+  "Past Sessions",
   "cards from",
 ];
 export const validCardFilter = ["best", "worst", "random", "oldest", "newest"];
```

Comment:
Maybe this can be refactored as enum?

Reply:
I was just thinking that it would be useful to leave it as is, so when we create the tokenizer, we can just pass in allTokens (which contains all the keywords) very easily